model-hub-catalog-v2.yaml

```diff
@@ -0,0 +1,718 @@
+# The model spec is designed keeping in mind both the HF and NGC APIs and tried to capture minimal information from both the APIs.
+models:
+  - name: llama3 instruct 
+    displayName: Llama 3 
+    description: Llama-3 instruct model published by Meta
+    modelVariants:
+      - variantId: llama 3 8b instruct
+        displayName: 8b
+        source:
+          URL: https://catalog.ngc.nvidia.com/orgs/nim/teams/meta/models/llama3-8b-instruct
+          type: NGC
```

Comment:
Can we move type to the top level? There's no need to have type for each modelVarant